README.md

```diff
@@ -13,12 +13,15 @@ Create a OpenStack environment
    When images are uploaded you'll need to: `Volumes` -> `Create Volume` -> `Volume Source` choose `Image` -> `Select image`. 
    
 2. Create `vps-ssd-3` instance from this volume: (`Boot Source` -> `Volume` -> `Your root volume`). 
-3. Go to info page and note the instance public key, ssh to the instance and check whether public key matches, 
-   save public key to your ssh config.
-4. Create a `hosts` file containing: 
+
+3. Create an additional blank volume for storing log and database dumps; we recommend 100GB. Attach it to the instance you created, and make a note of the device identifier it gets assigned (for example, `/dev/vdb` or `/dev/vdc`).
+
+4. Go to info page and note the instance public key, ssh to the instance and check whether public key matches, save public key to your ssh config.
```

Comment:
* Isn't it the private key that you need to save locally to ssh into the new instance?
* You can either save the key to your ssh config, or pass it to [`ansible-playbook --private-key`](https://www.mankier.com/1/ansible-playbook#--private-key), just like `ssh -i`.